portfolio/src/main/java/com/google/sps/data/Task.java

```diff
@@ -13,32 +14,55 @@
   private final Optional<String> description;
   private final Duration duration;
   private final TaskPriority priority;
+  private Instant scheduledTime;
 
   // TODO(raulcruise): Add an ID field if necessary.
 
   /**
    * The constructor will make sure that all necessary parameters are passed in, and populate each
    * class variable appropriately. None of the fields can be null with exception to the description
    * as it is optional. If the description is passed in as null then it will be stored as an empty
-   * Optional object.
+   * Optional object. Tasks by default do not have a scheduled time, they are only assigned this
+   * field as a result of getting scheduled by the scheduling algorithm.
    */
-  public Task(String name, String description, Duration duration, TaskPriority priority) {
+  public Task(String name, String description, long durationMinutes, TaskPriority priority) {
```

Comment:
I think we should try to stick with using `Duration` here. Why do we need to use `long` instead?

Reply:
I reverted back to Duration.